src/test/ui/use/use-after-move-self.rs

```diff
@@ -1,3 +1,4 @@
+//compile-flast: -Z borrowck=mir
```

Comment:
```suggestion
//compile-flags: -Z borrowck=mir
```

Reply:
Oops, didn't notice this. I think I'd prefer either removing the `// compile-flags` and letting the compare mode handle it (as it is currently with the misspelt comment) or adding an explicit `#![feature(nll)]`.